scenarios/kubernetes_e2e.py

```diff
@@ -396,13 +401,18 @@ def set_up_aws(workspace, args, mode, cluster, runner_args):
     if args.aws_cluster_domain:
         cluster = '%s.%s' % (cluster, args.aws_cluster_domain)
 
+    # AWS requires a username (and it varies per-image)
+    ssh_user = args.kops_ssh_user
+    if ssh_user == '':
+        ssh_user = 'admin'
+
     runner_args.extend([
         '--kops-cluster=%s' % cluster,
         '--kops-zones=%s' % zones,
         '--kops-state=%s' % args.kops_state,
         '--kops-nodes=%s' % args.kops_nodes,
         '--kops-ssh-key=%s' % aws_ssh,
-        "--kops-ssh-user=admin",
+        '--kops-ssh-user=%s' % ssh_user,
```

Comment:
Doesn't this first require a new kubekins image? Otherwise I think kubetest will blow up.

Reply:
I'd assume this works with the existing flag :-)

Reply:
I think I've (now) split this into separate PRs that won't break.

1: Set kops-ssh-user flag, default to admin.  We auto-export the env var KUBE_SSH_USER for kops, but that is correct I believe.  (We were setting it where it was not `admin`, and now we are setting it even when we are `admin`.  But `admin` was the default anyway, so we're just being more explicit).

2: #8787 stops setting the env vars, which will need an image update, so I'm going to hold off on that one.